pkg/solana/logpoller/discriminator.go

```diff
@@ -0,0 +1,14 @@
+package logpoller
+
+import (
+	"crypto/sha256"
+	"fmt"
+)
+
+const DiscriminatorLength = 8
+
+func Discriminator(namespace, name string) [DiscriminatorLength]byte {
```

Comment:
@ilija42 has a PR that also adds a discriminator: [here](https://github.com/smartcontractkit/chainlink-solana/pull/1014)

Reply:
Yeah, that PR will go in after this one... this will use his discriminator once that gets merged.